main.bicep

```diff
@@ -0,0 +1,8 @@
+resource stg 'Microsoft.Storage/storageAccounts@2019-06-01' = {
+    name: 'CHANGEME_TO_A_UNIQUE_NAME'
```

Comment:
what do you think about change the value of name to use the uniqueString() function?

```
name: uniqueString(resourceGroup().id)
```

Reply:
Great shout - have updated to use this.